windows-wwiv-update/Form2.cs

```diff
@@ -200,16 +202,44 @@ void m_oWorker_DoWork(object sender, DoWorkEventArgs e)
                 // Update Progress Bar
                 m_oWorker.ReportProgress(60);
 
+                // Create Temp Update Directory
+                Directory.CreateDirectory("update-temp");
+
+                // Create Cleanup Batch File If Not Existant
+                string cleanupFile = Path.Combine(Application.StartupPath, "cleanup.bat");
+                if (!File.Exists(cleanupFile))
+                {
+                    File.Delete(cleanupFile);
```

Comment:
You just checked that it didn't exist. Why delete it now? If the file doesn't exist already, you will never created it. Is that expected?  Also cleanup.bat is a generic name, why not something like wwiv-update-cleanup.cmd (use cmd not bat too since we don't want this to run on dos, right?)